package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@charmverse/core",
-  "version": "0.2.1",
+  "version": "0.2.2-rc-permissions-spa.1",
```

Comment:
this should be reverted right

Reply:
Not sure how this versioning works, I pulled from main and merged. Will revert to previous version (rc-permissions)